pycti/api/opencti_api_client.py

```diff
@@ -440,126 +441,107 @@ def not_empty(self, value):
         else:
             return False
 
-    def process_multiple(self, data: dict, with_pagination=False) -> Union[dict, list]:
-        """processes data returned by the OpenCTI API with multiple entities
-
-        :param data: data to process
-        :param with_pagination: whether to use pagination with the API
-        :returns: returns either a dict or list with the processes entities
+    def process_multiple(
+        self,
+        data: AnyDict,
+        with_pagination: bool = False,
+    ) -> Union[AnyDict, List[AnyDict]]:
         """
+        Process data returned by the OpenCTI API with multiple entities by flattening
+        a connection { edge { node { T } } into a T[]
+        :param data: Data to process
+        :param with_pagination: Whether to use pagination with the API
+        :return: A list of processed entities, inside a dict if with_pagination was enabled
+        """
+
+        if data is None:
+            data = {}
 
         if with_pagination:
             result = {"entities": [], "pagination": {}}
+            edges = data.get("edges") or []
+            for edge in edges:
+                node = edge["node"]
+                result["entities"].append(self.process_multiple_fields(node))
+                result["pagination"] = data["pageInfo"]
         else:
             result = []
-        if data is None:
-            return result
-        for edge in (
-            data["edges"] if "edges" in data and data["edges"] is not None else []
-        ):
-            row = edge["node"]
-            if with_pagination:
-                result["entities"].append(self.process_multiple_fields(row))
-            else:
-                result.append(self.process_multiple_fields(row))
-        if with_pagination and "pageInfo" in data:
-            result["pagination"] = data["pageInfo"]
+            edges = data.get("edges") or []
```

Comment:
```suggestion
            edges = data.get("edges", [])
```
Also possible like this ;)

Reply:
if for whatever reason there's an edges=None in the mapping, it'll explode. Now that shouldn't happen, but you never know sometimes :p